macros/miscs/startup.cfg

```diff
@@ -12,34 +12,35 @@ gcode:
 
     RESPOND MSG="Klippain started and ready !"
 
-
-# TMC2240 extruder initialization macro to set the driver registers correctly. See Esoterical github for more informations:
-# https://github.com/Esoterical/voron_canbus/blob/main/toolhead_flashing/common_hardware/BigTreeTech%20SB2209%20and%20SB2240/SB2240%20Stepper%20Configuration%20and%20Undervoltage.md
 [gcode_macro _INIT_TMC2240]
 gcode:
-    # Enable accurate stall current measurement
-    SET_TMC_FIELD STEPPER=extruder FIELD=pwm_meas_sd_enable VALUE=1
-    SET_TMC_FIELD STEPPER=extruder FIELD=sg4_filt_en VALUE=1
-    # Set the StealthChop stall detection threshold (may not be completely necessary)
-    SET_TMC_FIELD STEPPER=extruder FIELD=SG4_THRS VALUE=10
-    # Set the hold current to zero, and completely switch off the motor when it is not in use
-    SET_TMC_FIELD STEPPER=extruder FIELD=IHOLD VALUE=0
-    SET_TMC_FIELD STEPPER=extruder FIELD=freewheel VALUE=1
-    # Set the max expected velocity to a value such that we are unlikely to switch to fullstepping except during a very fast retraction or prime
-    SET_TMC_FIELD STEPPER=extruder FIELD=THIGH VELOCITY=50
-    # Use CoolStep, but we need a certain step frequency for it to work
-    SET_TMC_FIELD STEPPER=extruder FIELD=TCOOLTHRS VALUE=4000
-    # But do switch to PWM autotuning when at high flow
-    SET_TMC_FIELD STEPPER=extruder FIELD=TPWMTHRS VELOCITY=1
-    # Allow the motor to freewheel when not in use, means it runs cooler
-    SET_TMC_FIELD STEPPER=extruder FIELD=freewheel VALUE=1
-    # Set the temperature prewarning to something reasonable. Cosmetic, Klipper does nothing with this
-    SET_TMC_FIELD STEPPER=extruder FIELD=OVERTEMPPREWARNING_VTH VALUE=2885 # 7.7 * 100 C + 2038
-    # The following is absolutely critical: set the overvoltage snubber to a sensible voltage.
-    # This should be set to about 0.8 V above your power supply's idle voltage.
-    # Your PSU voltage can be read from the TMC 2240 by issuing a GCODE command:
-    # DUMP_TMC stepper=extruder register=ADC_VSUPPLY_AIN
-    # The voltage is the value of adc_vsupply multiplied by 0.009732
-    # It is also possible to use adc_vsupply + 82 here, which works out to be the same.
-    {% set v = (24.7/0.009732)|int %}
-    SET_TMC_FIELD STEPPER=extruder FIELD=OVERVOLTAGE_VTH VALUE={ v }
+    RESPOND MSG="You have defined to have a TMC2240 installed. If you need specific gcode settings, please override macro '_INIT_TMC2240'"
```

Comment:
This should not use RESPOND as it will spam user about that on each restart. Maybe just a comment is enough to say that these defaults fixes things based on Esoterical github but are not necessary for everyone and could be tweaked